docs/model_developers/modules.rst

```diff
@@ -309,6 +309,7 @@ to consider it released.
 You can package a v2 module with ``inmanta module build`` which will create a Python wheel.
 You can then publish this to the Python package repository of your choice,
 for example the public PyPi repository.
+The module with name <module-name> will be distributed as a Python package with name inmanta-module-<module-name>.
```

Comment:
I think it might be even more clear to give an example instead of using `<module-name>`, mainly to highlight the `-` vs `_`. Something along the lines of "The inmanta module ``my_module`` will be packaged as a Python package ``inmanta-module-my-module``." What do you think?

Reply:
You are right, it will remove all ambiguity about the _ becoming a -.